scripts/e2e/cmd/runner/testdata/one-namespace-one-ingress/three-namespaces/deploy.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+set -auexo pipefail
+
+echo -e "The goal of this is to ensure that containers with the same probel and same labels in 3 different namespaces have unique and working health probes"
+
+for ns in ns-x ns-y ns-z; do
+    kubectl create namespace "${ns}" || true
+
+kubectl apply -f - <<EOF
+apiVersion: apps/v1
```

Comment:
you can use helm instead

Reply:
we don't usually use the script, but It's handy when deploy it locally :)

Reply:
kubectl apply -f app.yaml ?

Reply:
> kubectl apply -f app.yaml ?

updated